scripts/get-model-data.py

```diff
@@ -286,14 +401,16 @@
             if (k[-1] if isinstance(k, (tuple, list)) else k) in data
         ]
         data = data[keys]
-
+        if 'process_fields' in config:
+            data = queries.process_fields(config['process_fields'], data)
         if 'rename_fields' in config:
             data = queries.rename_fields(config['rename_fields'], data)
 
         # sort list by some dict value
         if 'sort_by' in config:
             data = queries.sort_by(config['sort_by'], data)
 
+        data = data.drop_duplicates()
```

Comment:
Do we test this anywhere? I'd like to understand what sort of duplicates we might get as a result of the joins (we shouldn't get duplicates for straightforward model dumps, right?).

Reply:
Hey, no we shouldn't get duplicates from the standard model dumps (which is why this is safe here). And no this isn't tested anywhere. To test it I ran the csv generation locally using master and this branch and made sure the respective  csvs had the same number of rows. 

This new csv does have duplicates. It lists briefs but is actually created from the brief_responses csv.
We have to load up all the brief responses to get the counts for how many applied so until we do the drop duplicates there is a row for every brief response. 

At this point in the code the brief response rows are exact duplicates because we have filtered out columns that are brief response specific using the `keys` section of the config.

Doing this at the end is like doing a sql `SELECT DISTINCT ON`

Bit fiddly! 😅

Reply:
Cool, that makes sense. However in the long term I think I'd feel better if `drop_duplicates` was only done for this new dump or ones like it - maybe add a comment or a note to the tech debt ticket to make it clear why we might get duplicates in the first place? (Your explanation above would do just fine!)